parts_list/master_parts_list_raw.csv

```diff
@@ -1,144 +1,146 @@
-#4-40 x 1/2 Threaded Standoff,T5,91780A164,McMaster,https://www.mcmaster.com/91780A164,1,1,1,$0.27 ,$0.27,
-#4-40 x 3/4 Threaded Standoff,T6,91780A166,McMaster,https://www.mcmaster.com/91780A164,1,1,1,$0.37 ,$0.37,
-M2.5 x 12mm Standoff,T7,95947A007,McMaster,https://www.mcmaster.com/95947A007,1,1,1,$0.64 ,$0.64,
-"#2-56 x 3/8"" Threaded Standoff",T8,93330A252,McMaster,https://www.mcmaster.com/93330A252,1,4,4,$1.00 ,$4.00,PCB Assembly
-M2.5 x 10 mm Standoff,T10,95947A005,McMaster,https://www.mcmaster.com/95947a005,1,8,8,$0.59 ,$4.72,"Head Assembly, PCB Assembly"
-"#2-56 x 1/2"" Standoff",T11,91780A032,McMaster,https://www.mcmaster.com/91780a032,1,20,20,$0.41 ,$8.20,PCB Assembly
-#4-40 Heat Set Inserts,I1,93365A120,McMaster,https://www.mcmaster.com/93365a120,100,8,1,$10.82 ,$10.82,Head Assembly
-#6-32 Heat Set Inserts,I2,93365A130,McMaster,https://www.mcmaster.com/93365a130,100,4,1,$12.92 ,$12.92,Body
-Allen Key Set,D2,,,,,1,,,,"Wheel Assembly, Rocker-Bogie, Corner Steering, Body, Mechanical Integration, Differential Pivot"
-Center punch or Starter drill bit,D7,,,,,1,,,,"Wheel Assembly, Differential Pivot"
-Hand Drill or Drill Press,D3,,,,,1,,,,"Wheel Assembly, Differential Pivot"
-Drill bit \#23,D6,,,,,1,,,,"Wheel Assembly, Differential Pivot"
-Metal Hacksaw or Bandsaw,D4,,,,,1,,,,"Rocker-Bogie, Head Assembly, Wheel Assembly, Differential Pivot"
-Vice clamp or C Clamps,D5,,,,,1,,,,"Wheel Assembly, Differential Pivot"
-5/16 Wrench,D1,,,,,1,,,,"Rocker-Bogie, Mechanical Integration, Wheel Assembly, Differential Pivot"
-Vice or V-Clamp,D8,,,,,1,,,,"Head Assembly, Differential Pivot"
-YellowJacket 223 rpm drive motor,E46,5202-0002-0027,GoBilda,https://www.gobilda.com/5202-series-yellow-jacket-planetary-gear-motor-26-9-1-ratio-223-rpm-3-3-5v-encoder/,1,6,6,$39.99,$239.94,Wheel Assembly
-M4 x 0.7mm 6mm screws,B15,2802-0004-0006,GoBilda,https://www.gobilda.com/2802-series-zinc-plated-steel-button-head-screw-m4-x-0-7mm-6mm-length-25-pack/,25,48,2,$1.79,$3.58,Wheel Assembly
-GoBilda 2 hole U-Channel,S45,1120-0002-0072,GoBilda,https://www.gobilda.com/1120-series-u-channel-2-hole-72mm-length/,1,6,6,$2.79,$33.48,Wheel Assembly
-GoBilda 23mm threaded plate,S46,2803-0023-0023,GoBilda,https://www.gobilda.com/2803-series-stainless-steel-threaded-plate-23-23/,1,12,12,$2.49,$29.88,Wheel Assembly
-2-post 43-32mm drive motor clamping mount,S47,1401-0043-0032,GoBilda,https://www.gobilda.com/1401-series-2-side-2-post-clamping-mount-43mm-width-32mm-bore/,1,6,6,$5.99,$35.94,Wheel Assembly
-GoBilda Pattern Adaptor,S48,1206-0016-0004,GoBilda,https://www.gobilda.com/1206-series-pattern-adaptor-16-4/,1,12,12,$4.99,$59.88,Wheel Assembly
-6mm Bore Clamping D-Hub 0.770in,S49,545618,ServoCity,https://www.servocity.com/6mm-bore-clamping-d-hub-tapped-0-770-pattern/,1,6,6,$7.99,$47.94,Wheel Assembly
+"Part Name","Project Ref Code","Model/Config","Site","Link","Sold in Packs of","Total Used in Project","Quantity to Buy","Price Each","Price Total","Used in Sections"
```

Comment:
csv file needs quotation marks removed from everywhere, impossible to tell what changed from the diff

Reply:
I have fixed the quotation mark problem.